AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_entity_utils.py

```diff
@@ -473,12 +473,11 @@ def remove_components(self, component_names: list):
         :param component_names: List of component names to remove
         :return: None
         """
-        type_ids = EditorComponent.get_type_ids(component_names, EditorEntityType.GAME)
-        for type_id in type_ids:
-            remove_outcome = editor.EditorComponentAPIBus(bus.Broadcast, "RemoveComponents", self.id, [type_id])
-            assert (
-                remove_outcome.IsSuccess()
-            ), f"Failure: could not remove component from '{self.get_name()}'"
+        component_ids = [component.id for component in self.get_components_of_type(component_names)]
+        remove_success = editor.EditorComponentAPIBus(bus.Broadcast, "RemoveComponents", component_ids)
+        assert (
+            remove_success
+        ), f"Failure: could not remove component from '{self.get_name()}'"
```

Comment:
shouldn't `'{self.get_name()}'` be `'{self.get_components_of_type(component_names)}'`?

Reply:
self.get_components_of_type will return a list of EntityComponent which will not indicate anything here. self.get_name() returns the entity name from which a component could not be removed. There's no indication which component would have failed or why. I could clarify: from entity '{self.get_name()}'